test/integration/rtpbroadcast.js

```diff
@@ -34,7 +35,8 @@ describe('Rtpbroadcast tests', function() {
       .then(function(session) {
         janusSession = session;
         done();
```

Comment:
oh! did know that `done()` [can take exception in param](https://mochajs.org/#asynchronous-code), but in this case, according to the doc, you can even return the promise without calling `done()` method at all I guess...